README.md

```diff
@@ -19,7 +19,7 @@ Example:
 
 ### Logo
 
-You can add your logo under ``static/images/members``. You can choose between different file types (jpg, gif, png, svg) but we recomment you to use svg. Please use a quarter icon with 1:1. 
+You can add your logo under ``static/images/members``. You can choose between different file types (jpg, gif, png, svg) but we recomment you to use svg. Please use a square logo with the 1:1 dimming
```

Comment:
I am unsure what dimming means.

Reply:
other than that, this looks good to go

Reply:
i correct this, sorry was late! i mean a ratio by 1:1. nothing more than a square ;-)